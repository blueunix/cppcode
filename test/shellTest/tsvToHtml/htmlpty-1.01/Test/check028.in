<!-- From: http://ojps.aip.org/journal_cgi/dbt?KEY=JMAPAQ&Volume=40&Issue=10 -->
<!-- (Journal of Mathematical Physics v40 n10 October 1999) -->
<strong>The effective <img border=0 src="/stockgif2/sgr.gif" alt="<i>sigma</i>" align="bottom">-model of multidimensional gravity</strong><br>
