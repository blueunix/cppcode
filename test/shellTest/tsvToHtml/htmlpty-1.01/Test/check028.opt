-brief -no-comment-banner -quiet -indent 2
