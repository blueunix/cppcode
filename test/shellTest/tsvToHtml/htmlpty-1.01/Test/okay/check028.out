<!-- From: http://ojps.aip.org/journal_cgi/dbt?KEY=JMAPAQ&Volume=40&Issue=10 -->
<!-- (Journal of Mathematical Physics v40 n10 October 1999) -->
<STRONG>The effective <IMG BORDER=0 SRC="/stockgif2/sgr.gif" ALT="<i">
sigma</I> &quot; align=&quot;bottom&quot;&gt;-model of multidimensional
gravity</STRONG><BR>
